### ISSUE TITLE
ImageNet21K data preparation for pre-training

### ISSUE THREAD

**Issue by mxjecho** [facebookresearch/deit#219]
Hi,
I want to try imagenet21k pretraining. Could you please tell me how to process the raw data of imagenet21k?

**Comment by TouvronHugo** [comment]
Hi @mxjecho,
Thanks for your message.
You mean which data augmentation used?
Best,
Hugo

**Comment by mxjecho** [comment]
No, I want to know if I need to do pre-processing on the dataset. I use the winter21 imagenet 21k dataset.

**Comment by TouvronHugo** [comment]
We didn't do any particular preprocessing, we re-use the version we have internally of the dataset which should corresponds to the official version of the ImageNet dataset

**Comment by mxjecho** [comment]
Get it. BTW, what is the 'num_class' of Imagenet 21k?

**Comment by TouvronHugo** [comment]
21844 if I'm not wrong
